Makefile

```diff
@@ -367,7 +367,7 @@ check-capi: capi-setup
 		--no-default-features --features wat,compiler,wasi,middlewares $(capi_compiler_features)
 
 build-wasmer:
-	$(CARGO_BINARY) build $(CARGO_TARGET) --release --manifest-path lib/cli/Cargo.toml $(compiler_features) --features="webc_runner" --bin wasmer
+	$(CARGO_BINARY) build $(CARGO_TARGET) --release --manifest-path lib/cli/Cargo.toml --features cranelift,singlepass --features="webc_runner" --bin wasmer
```

Comment:
Mmm, that shuld not stay like that right?

Reply:
yeah, it was just for testing, because my Linux VM doesn't support building with LLVM

Reply:
Reverted in https://github.com/wasmerio/wasmer/pull/3426/commits/8b83cfcec88dea195595a6ff58d304d62c9db67b